ddm_test_validate.py

```diff
@@ -0,0 +1,214 @@
+import logging
+import datetime
+import pathlib
+import os
+import time
+from distutils.util import strtobool
```

Comment:
also unused

Reply:
`strtobool` is unused, removing that one. The others are same as in above comment.